src/generator.cpp

```diff
@@ -869,15 +815,19 @@ std::string Generator::GeneratorImpl::generateOperatorCode(const std::string &op
                                                            const AnalyserEquationAstPtr &ast) const
 {
     // Generate the code for the left and right branches of the given AST.
+    // Note: see the llvm-cov section in src/README.rst for astLeftChild and
+    //       astRightChild.
 
+    std::string res;
     auto astLeftChild = ast->leftChild();
     auto astRightChild = ast->rightChild();
     auto astLeftChildCode = generateCode(astLeftChild);
     auto astRightChildCode = generateCode(astRightChild);
 
     // Determine whether parentheses should be added around the left and/or
-    // right piece of code, and this based on the precedence of the operators
-    // used in CellML, which are listed below from higher to lower precedence:
+    // right piece of code, and this based on the precedence of the
+    // operators used in CellML, which are listed below from higher to lower
+    // precedence:
```

Comment:
Not sure about the width being used here for comments.  This has changed solely due to  a width setting of the user.

Reply:
That code was modified at some point and unmodified and I missed that bit. Done in 6a68a501.